llarp/service/endpoint.cpp

```diff
@@ -1770,6 +1772,9 @@ namespace llarp
     bool
     Endpoint::OutboundContext::Tick(llarp_time_t now)
     {
+      // we are probably dead af
+      if(m_LookupFails > 16)
+        return true;
```

Comment:
should we tell the operator? like log something? Otherwise we'll have HS just go offline and not know why?